File: mobile/src/app/repository/invitation/invitationDto.ts
import {UserDtoShort} from '../user/userDtoShort';

export class InvitationDto {

    public idInvitation: number;
    public senderInvitation: UserDtoShort;
    public receiverInvitation: UserDtoShort;
    public dateInvitation: string;

}

File: backend/src/main/java/com/wildBirds/BlueChat/api/rest/dto/InvitationDto.java
package com.wildBirds.BlueChat.api.rest.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@NoArgsConstructor
public class InvitationDto {

    private Long idInvitation;
    private UserDtoShort senderInvitation;
    private UserDtoShort receiverInvitation;
    private Instant dateInvitation;
}
